GNUmakefile

```diff
@@ -2,15 +2,13 @@ TEST?=$$(go list ./...)
 WEBSITE_REPO=github.com/hashicorp/terraform-website
 PKG_NAME=google
 
-GO111MODULE=on
```

Comment:
I'm so sorry. As I'm making these changes in beta, I saw this and I wonder if it does any harm leaving this in? I'm thinking in case the current environment already has it set, it might be nice to have this here to override it specifically for this makefile. What do you think?

Reply:
Hmm, I don't think there are too many people who have `GO111MODULE=off` in their shell configs. That would break too many things. The only time I've seen in set to `off` is when you're want the old `go get` behavior, e.g.

```sh
GO111MODULE=off go get github.com/foo/bar/cmd/baz
```

I could be wrong, though.

My preference is to embrace the new default behavior, and let people disable it inline (like the snippet above), but I'll leave it up to you.

Reply:
Ok!  We'll try it and if we get any feedback we can always add it back in.  That sounds find to me! Thanks!